style.css

```diff
@@ -0,0 +1,165 @@
+body {
+	width: 100%;
+	height: 100%;
+	margin: 0 auto;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/evagame/pullRequest?prid=5788637)